steamcmd_servers/suicidesurvival/egg-suicide-survival.json

```diff
@@ -0,0 +1,95 @@
+            "container": "debian:buster-slim",
+            "entrypoint": "bash"
+        }
+    },
+    "variables": [
+        {
+            "name": "APP ID",
+            "description": "The ID corresponding to the game to download.",
+            "env_variable": "APPID",
+            "default_value": "310",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|numeric|digits_between:1,6"
+        },
+        {
+            "name": "Maximum Players",
+            "description": "",
+            "env_variable": "MAX_PLAYERS",
+            "default_value": "12",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|numeric|digits_between:1,3"
+        },
+        {
+            "name": "WINEARCH",
+            "description": "WINEARCH. win32 or win64",
+            "env_variable": "WINEARCH",
+            "default_value": "win64",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|in:win32,win64"
+        },
+        {
+            "name": "WINEPREFIX",
+            "description": "WINEPREFIX",
+            "env_variable": "WINEPREFIX",
+            "default_value": "\/home\/container\/.wine",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "required|string|max:64"
+        },
+        {
+            "name": "WINEDEBUG",
+            "description": "WINEDEBUG: Debug Level of wine",
+            "env_variable": "WINEDEBUG",
+            "default_value": "-all",
+            "user_viewable": false,
+            "user_editable": false,
+            "rules": "nullable|string|max:20"
+        },
+        {
+            "name": "Map",
+            "description": "",
+            "env_variable": "MAP",
+            "default_value": "ss_garden2",
+            "user_viewable": true,
+            "user_editable": true,
+            "rules": "required|string|max:50"
+        },
+        {
+            "name": "Ip",
+            "description": "the IP address to use when starting wine",
+            "env_variable": "IP",
+            "default_value": "0.0.0.0",
+            "user_viewable": true,
+            "user_editable": true,
```

Comment:
This too. Otherwise users will not be able to start their server, when entering the wrong things.